src/Build/Logging/FancyLogger/FancyLoggerNode.cs

```diff
@@ -4,148 +4,136 @@
 
 using System;
 using System.Collections.Generic;
+using System.IO;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 using Microsoft.Build.Framework;
 
 namespace Microsoft.Build.Logging.FancyLogger
 { 
-    // TODO: Maybe remove
-    public enum FancyLoggerNodeType
+    public class FancyLoggerProjectNode
```

Comment:
nit:
Do all these things need to be public? They feel like internal implementation details.